lib/storage/release.go

```diff
@@ -129,6 +135,16 @@ func (r *ReleaseV1) GetChartVersion() string {
 	return r.Spec.ChartVersion
 }
 
+// GetChartIcon returns the chart application icon.
+func (r *ReleaseV1) GetChartIcon() string {
+	return r.Spec.ChartIcon
+}
+
+// SetChartIcon returns the chart application icon.
```

Comment:
~~returns~~ sets the icon to the specified value.
Btw, is ChartIcon the actual icon (base64-encoded?) or a reference to the external resource?

Reply:
It can be either, but in our use-case it is most often the actual b64-encoded icon data that comes from the manifest's "logo" field.